python/ray/tune/tune.py

```diff
@@ -457,7 +457,8 @@ def run_experiments(experiments,
                     reuse_actors=False,
                     trial_executor=None,
                     raise_on_failed_trial=True,
-                    concurrent=True):
+                    concurrent=True,
+                    callbacks=None):
```

Comment:
Seems fine to leave it (or remove), but don't forget to docstring

Reply:
Hm, the `run_experiments` doesn't seem to have a docstring at all. It is documented at `run()` though, like the other arguments. I can add a docstring to `run_experiments` if that'd help